### ISSUE TITLE
Why sometimes not all DNS records are pulled using api?

### ISSUE THREAD

**Issue by snyxan** [cloudflare/python-cloudflare#122]
I have hundreds of DNS records in cf, if I use python api only a subset of DNS records are returned.

I'm using something like this:
```
zones = cf.zones.get()
for zone in zones:
    zone_id = zone['id']
    dns_records = cf.zones.dns_records.get(zone_id)
    for record in dns_records:
        print(record['name'])
```

**Comment by mahtin** [comment]
The API only returns a finite number of entries. Please use the paging method - see **README**.

You can't see Python code to do similar paging in https://github.com/cloudflare/python-cloudflare/blob/master/examples/example_paging_thru_zones.py

You could cheat and set `per_page` to a very large number; but that will come back snd bite you one day.

Hope that helps.

PS: Read **Pagination** section in https://api.cloudflare.com/#getting-started-requests

**Comment by alfredocambera** [comment]
Here is my python gist to export all the zones and records into CSV format https://gist.github.com/alfredocambera/6349b363750beabdf73683bd6c1d94db

**Comment by mahtin** [comment]
In your code you're saying `page': 1}` vs using a page number variable that's incremented by `1` each loop. Ie. Have `page_no += 1` at the end of the loop and `page': page_no}` in the call.

There's an example of how to cone this in the `examples` folder (and I believe in the README).

**Comment by alfredocambera** [comment]
@mahtin Thanks for reviewing my gist 👍 

I updated the code to:
- Use `range` starting on 1 as it throws an exception if I start with 0 (`CloudFlare.exceptions.CloudFlareAPIError: Page must be a number greater than 0`) as in the zones. Also added +1 to the total so it counts all the pages correctly.
- Fix the header printing once per zone and not once at the top of the output.

Just checked and the total number of returned records per zone is correct. Let me know if you see any other improvement opportunity on my script to fix it.

Regarding the examples I found this one https://github.com/cloudflare/python-cloudflare/blob/master/examples/example_zones.py#L52-L69 but could not find examples on how to paginate over records.
